contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -596,6 +596,16 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
         return _checkIsAllowedRelayer(delivererAddress, allowedRelayers);
     }
 
+    /**
+     * @dev Sets the value of `blockchainID` to the value determined by the warp messenger precompile.
+     */
+    function initializeBlockchainID() public {
```

Comment:
can we name this `initialize()`? Follows what is more commonly used, and `initializeBlockchainID` sounds to me we're just initializing for one field, but doesn't imply intiailizing for the contract

Reply:
I think `initializeBlockchainID` is more descriptive in this case in this case we are just initializing one field and nothing else about the contract. For instance, you can send messages prior to the blockchain ID being initialized, and message IDs and everything else will be set/incremented.

Reply:
hmm true with the `receiveCrossChainMessage` lazy initialization, we don't really even need a separate transaction to `initializeBlockChainID`.